src/providers/fusionauth.ts

```diff
@@ -0,0 +1,52 @@
+import { OAuthConfig, OAuthUserConfig } from "./oauth"
+
+// This is the default openid signature returned from FusionAuth
+// it can be customized using lambda functions https://fusionauth.io/docs/v1/tech/lambdas/
+export interface FusionAuthProfile {
```

Comment:
```suggestion
/** This is the default openid signature returned from FusionAuth
 * it can be customized using [lambda functions](https://fusionauth.io/docs/v1/tech/lambdas)
 */
export interface FusionAuthProfile {
```